tools/integration_tests/util/operations/dir_operations.go

```diff
@@ -60,7 +61,7 @@ func RenameDir(dirName string, newDirName string) (err error) {
 }
 
 func CreateDirectoryWithNFiles(numberOfFiles int, dirPath string, prefix string, t *testing.T) {
-	err := os.Mkdir(dirPath, FilePermission_0600)
+	err := os.Mkdir(dirPath, FilePermission_0777)
```

Comment:
local directory does not give permission to move files from directory to bucket with 0600 permission. This permission is working with all tests.

Reply:
you can use `gsutil cp`

Reply:
for that also it is having 
``` dir_operations.go:74: Create file at "/home/tulsishah_google_com/directoryWithTwelveThousandFiles": open /home/tulsishah_google_com/directoryWithTwelveThousandFiles/fileInDirectoryWithTwelveThousandFiles1: permission denied```